packetbeat/protos/dns/dns_test.go

```diff
@@ -649,3 +788,266 @@ func assertAddress(t testing.TB, expected common.IpPortTuple, endpoint interface
 	assert.Equal(t, expected.Src_ip.String(), e.Ip)
 	assert.Equal(t, expected.Src_port, e.Port)
 }
+
+// TCP tests
+
+func testTcpTuple() *common.TcpTuple {
+	t := &common.TcpTuple{
+		Ip_length: 4,
+		Src_ip:    net.IPv4(192, 168, 0, 1), Dst_ip: net.IPv4(192, 168, 0, 2),
+		Src_port: ClientPort, Dst_port: ServerPort,
+	}
+	t.ComputeHashebles()
+	return t
+}
+
+// If a TCP gap (lost packets) happen while we're waiting for parts
```

Comment:
This would be a good test case to add. If we don't add it now, let's remove this commented out placeholder and add a comment describing the test case to the `TODO` header at the top of the file.